docs/server-side/v2/introduction/webhook.mdx

```diff
@@ -567,9 +577,11 @@ This event will be sent when final composed recording is generated and uploaded
 | template_id        | `string`             | Template ID of the room <br/><br/> _Example: 66112497abcd52312556c4gg _                                                            |
 | location           | `string`             | s3 URI of the recorded video <br/><br/> _Example: s3://bucket/prefix/ac.mp4 _                                                      |
 | URL                | `string`             | Presigned s3 URL for the recorded video, for download. Valid for 24 hours <br/><br/> _Example: https://s3-location/bucket/ac.mp4 _ |
-| duration           | `number`             | Duration the user spent in the room in seconds <br/><br/> _Example: 3600 _                                                         |
+| duration           | `number`             | Duration of recording in seconds <br/><br/> _Example: 3600 _                                                         |
+| session_duration   | `number`             | Duration the user spent in the room in seconds <br/><br/> _Example: 36000 _              |
```

Comment:
@atul107 Here, we're saying that session duration is in seconds but looks like we're showing it in mins in the sample payload below. Please check.

Reply:
session_duration should be int (seconds). The example should be valid/correct use case.

Reply:
@atul107 I see the example updated as 40 now. Couple of suggestions:

- Shall we update session_started_at and session_stopped_at fields, currently they have same values. 
"session_started_at": "2021-11-30T12:58:49.97291247Z",
"session_stopped_at": "2021-11-30T12:58:49.97291247Z"
- may be to show like a 10 mins difference so that we can show 600 seconds as value for session_duration (to emphasize this is in seconds, 40 seems very low).

Reply:
Updated `session_started_at` and `session_duration` values